app/tests/src/test_retrieve.py

```diff
@@ -3,11 +3,8 @@
 
 from src.db.models.document import Document
 from src.retrieve import retrieve_with_scores
-from tests.mock.mock_sentence_transformer import MockSentenceTransformer
 from tests.src.db.models.factories import ChunkFactory, DocumentFactory
 
-mock_embedding_model = MockSentenceTransformer()
```

Comment:
`mock_embedding_model`/`MockSentenceTransformer` is now provided by the `app_config` fixture via `MockAppConfig.sentence_transformer()`.